web_external/models/DatasetModel.js

```diff
@@ -327,6 +327,15 @@ const DatasetModel = MinervaModel.extend({
         this.set('visible', true);
         this.set('opacity', 1);
         this.set('displayed', false);
+    },
+
+    getBoundSupported() {
+        if (this.get('_id').toString().length < 7) {
```

Comment:
Where is an example of one of these short ids?

Reply:
Oh, thanks for your suggestion, I made a change with the following commit.